challenge.md

```diff
@@ -32,19 +32,19 @@ Open up this file in your text editor, and write your answers below (where indic
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
 <!-- Answer Starts Here -->
-
+Absolute path -- starts at "Users" so you know the computer will recognize it wherever you are.
 <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
 <!-- Answer Starts Here -->
-
+Absolute, so others can access it wherever they are, whatever machine they're on.
```

Comment:
Again, you're correct that it's absolute, but your explanation is a little fuzzy. Someone else hosted that font; we use an absolute path because, no matter where our project is, the location of the font will remain the same.